client/src/App.jsx

```diff
@@ -8,6 +8,9 @@ import {createBrowserHistory} from 'history';
 import urlHelpers from './urlHelpers.js';
 import availabilityHelpers from './availabilityHelpers';
 import Guests from './Guests.jsx';
+import AppStyles from './AppStyles.js';
```

Comment:
@rgagnon2 Assuming this file is coming, but I think this line could break your production build since it doesn't currently exist as of this PR.